config/configtelemetry/doc.go

```diff
@@ -4,4 +4,46 @@
 // Package configtelemetry defines various telemetry level for configuration.
 // It enables every component to have access to telemetry level
 // to enable metrics only when necessary.
+//
+// 1. configtelemetry.None
+//
+// No telemetry data should be collected.
+//
+// 2. configtelemetry.Basic
+//
+// Signals associated with this level cover the essential coverage of the component telemetry.
+//
+// This is the default level recommended when running the collector.
+//
+// The signals associated with this level must have low cardinality.
+//
+// The signals associated with this level must represent a small data volume:
+//   - No more than 5 metrics reported.
+//   - At most 1 span actively recording simultaneously, covering the critical path.
+//   - At most 5 log records produced every 30s.
+//
+// Not all signals defined in the component telemetry are active.
+//
+// 3. configtelemetry.Normal
+//
+// Signals associated with this level cover the complete coverage of the component telemetry.
+//
+// The signals associated with this level must control cardinality.
+//
+// The signals associated with this level must represent a controlled data volume:
+//   - No more than 20 metrics reported.
```

Comment:
is this 20 individual meters or 20 metric points?

Reply:
20 different metric time series, as in 20 combination of 20 metric names and dimension names values.

Reply:
Doesn't that mean a single metric that records the http status code would not fall into the normal level because the cardinality is >70?

Reply:
[http.server.request.duration](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-metrics.md#metric-httpserverrequestduration)

Reply:
Checking, I think indeed that metric would be too verbose. You can create quite a bit of cardinality if you are running a server and any URI or method adds to cardinality. But say for the httpcheckreceiver, that might be fine because you use a http client to call just one endpoint, so cardinality is fixed.

Reply:
IMO given the metric is labeled "required" this should be covered by the normal telemetry settings.

Reply:
ok, so let's relax this. Can you offer an amendment? Maybe we can set a max cardinality of 500 for now.
```suggestion
//   - A max cardinality (total possible combinations of dimension values) of 500.
```

Reply:
I think that's alright, I'd be interested in what enforcement looks like in the future given there's support for cardinality limits on instruments. https://github.com/open-telemetry/opentelemetry-go/blob/main/sdk/metric/internal/x/README.md#L11

Reply:
you probably get to write that part ;)